src/components/toastNotificationsList/toastNotificationsList.js

```diff
@@ -21,46 +21,41 @@ class ToastNotificationsList extends React.Component {
   };
 
   render() {
-    const { toasts, paused } = this.props;
+    const { toasts } = this.props;
+    const timeout = 8000;
```

Comment:
required:
convert hardcoded timeout to a `prop` with a `default` of `8000`... bit easier than having to figure out where that number is buried, and if we do switch to another method of providing that number bit easier to update

optional:
consider making this a dotenv parameter in `.env` maybe something like
```
REACT_APP_TOAST_NOTIFICATIONS_TIMEOUT=8000
```
but then you'd also need to bring it in through `./src/common/helpers.js` annnnnd now it's starting to move into refactor land

Reply:
dotenv params come across as strings, we just need to convert it towards an int... in helpers go ahead and add this wrapper... the `undefined` is there so you don't receive `NaN`
```
const TOAST_NOTIFICATIONS_TIMEOUT = Number.parseInt(process.env.REACT_APP_TOAST_NOTIFICATIONS_TIMEOUT, 10) || undefined;
```

Reply:
Thank you! Worked like a charm